packages/itmat-interface/src/server/router.ts

```diff
@@ -176,7 +177,10 @@ export class Router {
             target: config.aeEndpoint,
             changeOrigin: true,
             xfwd: true,
-            headers: { authorization: `Basic ${Buffer.from('kai:token').toString('base64')}` }
+            onProxyReq: function (preq, req:any){
```

Comment:
If you're going to be using the proxy request modifier can you then remove the `ae_proxy_preprocessor` at line 170 and put it here too. Otherwise we have two different things doing similar things are different places 👅